network-libp2p/src/connection_pool/behaviour.rs

```diff
@@ -83,6 +85,7 @@ impl Default for Config {
             ipv4_subnet_prefix_len: 24,
             ipv6_subnet_prefix_len: 96,
             dialing_count_max: 3,
+            num_initial_connections: 1,
```

Comment:
Why `1` here, when the default is `4`?

Reply:
This is the structures default. It is different from the config file default. If I would put a 4 here I would have to change it if the default changes here as well. 

This number is overwritten when we use it here: https://github.com/nimiq/core-rs-albatross/blob/nibhar/more-seed-nodes/network-libp2p/src/behaviour.rs#L90-L97

1 as the structures default did make sense to me as it is the minimum viable number and will also work for example in a test scenario where there is not even 3 total nodes to begin with.

It could be changed I guess. The default implementation could probably removed for example making only one spot the actual default.